CONTRIBUTORS.rst

```diff
@@ -7,19 +7,20 @@ Core Developers
 These contributors have commit flags for the repository,
 and are able to accept and merge pull requests.
 
-=========================== ================ ===========
-Name                        Github           Twitter
-=========================== ================ ===========
-Daniel Roy Greenfeld        `@pydanny`_      @pydanny
-Audrey Roy Greenfeld*       `@audreyr`_      @audreyr
-Fábio C. Barrionuevo da Luz `@luzfcb`_       @luzfcb
-Saurabh Kumar               `@theskumar`_    @_theskumar
+=========================== ================== ===========
+Name                        Github             Twitter
+=========================== ================== ===========
+Daniel Roy Greenfeld        `@pydanny`_        @pydanny
+Audrey Roy Greenfeld*       `@audreyr`_        @audreyr
+Fábio C. Barrionuevo da Luz `@luzfcb`_         @luzfcb
+Saurabh Kumar               `@theskumar`_      @_theskumar
 Jannis Gebauer              `@jayfk`_
-Burhan Khalid               `@burhan`_       @burhan
-Nikita Shupeyko             `@webyneter`_    @webyneter
-Bruno Alla                  `@browniebroke`_ @_BrunoAlla
-Wan Liuyang                 `@sfdye`_        @sfdye
-=========================== ================ ===========
+Burhan Khalid               `@burhan`_         @burhan
+Nikita Shupeyko             `@webyneter`_      @webyneter
+Bruno Alla                  `@browniebroke`_   @_BrunoAlla
+Wan Liuyang                 `@sfdye`_          @sfdye
+Curtis St Pierre            `@curtisstpierre`_ @cstpierre1388
```

Comment:
This list is the core maintainers who can merge pull requests and commit to the repo. All the contributors are listed in the following list, that where you should add your name.

Also, the second column is [a reStrusturedText hyperlink](http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#hyperlinks), you should also define its target somewhere in the file, we do that at the bottom of it, for instance:

https://github.com/pydanny/cookiecutter-django/blob/98012829f3c5a2dbfa7962f744590685513de956/CONTRIBUTORS.rst#L195

Reply:
Fixed in the newest commit, and added the link. Thank you!